pkg/edge/chromium_arm64.go

```diff
@@ -0,0 +1,26 @@
+//go:build windows
+// +build windows
+
+package edge
+
+import (
+	"unsafe"
+
+	"github.com/leaanthony/go-webview2/internal/w32"
+)
+
+func (e *Chromium) Resize() {
+	if e.controller == nil {
+		return
+	}
+
+	var bounds w32.Rect
+	w32.User32GetClientRect.Call(e.hwnd, uintptr(unsafe.Pointer(&bounds)))
+
+	words := (*[2]uintptr)(unsafe.Pointer(&bounds))
```

Comment:
It would be possible to access the bounds fields directly, but would need to combine left and top bit-shifted and the same for right/bottom.
This seemed to be an easy way to split the struct into words without thinking about endianness. It's also done like this in other projects, e.g. [wireguard](https://git.zx2c4.com/wireguard-windows/tree/tunnel/winipcfg/winipcfg.go?h=v0.3.8#n184)

Reply:
Yeah cool. That's great - thanks for the explaination!

Reply:
You're very welcome.